lib/openc_bot/helpers/dates.rb

```diff
@@ -25,8 +25,6 @@ def normalise_us_date(raw_date)
         to_date(iso_date, true).to_s
       end
 
-      private
```

Comment:
I'm interested to know a bit more about this. Any chance of a bit more description in the commit, or a link to something talking about it..?

Reply:
Found a better fix and an answer: https://stackoverflow.com/a/18518879/63349

Code and commit message updated.